azure_monitor/src/azure_monitor/sdk/auto_collection/live_metrics/exporter.py

```diff
@@ -0,0 +1,168 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+#
+import collections
+import logging
+import typing
+
+from opentelemetry.sdk.metrics import Counter, Observer
+from opentelemetry.sdk.metrics.export import (
+    MetricRecord,
+    MetricsExporter,
+    MetricsExportResult,
+)
+
+from azure_monitor.protocol import (
+    Envelope,
+    LiveMetric,
+    LiveMetricDocument,
+    LiveMetricDocumentProperty,
+    LiveMetricEnvelope,
+)
+from azure_monitor.sdk.auto_collection import live_metrics
+from azure_monitor.sdk.auto_collection.live_metrics.sender import (
+    LiveMetricsSender,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class LiveMetricsExporter(MetricsExporter):
+    """Live Metrics Exporter
+
+    Export data to Azure Live Metrics service and determine if user is subscribed.
+    """
+
+    def __init__(self, instrumentation_key):
+        self._instrumentation_key = instrumentation_key
+        self._sender = LiveMetricsSender(self._instrumentation_key)
+        self.subscribed = True
```

Comment:
I don't like the exporter keeping the state of subscribed, however there doesn't seem to be a nice way to do this. One way is to use the `context`, where the key is LIVE_METRICS_{x} where x is the unique UUID for that manager and the value is subscribed or not. Another (ugly) way is to have a reference to the manager from the exporter and keep the state of subscribed in the manager.

Reply:
Since UUID is unique per application and not manager instance, I think it is fine to have a global variable in the context keeping track of the state of subscribed. Thoughts?

Reply:
Problem is with internal threads, we have 3 different threads that would be using this global context variable, making this code easier to break in specific cases, my point is that is not worth to add this logic only because subscribed flag feels out of place.